core/pom.xml

```diff
@@ -587,6 +587,12 @@
 			<version>2.3.5</version>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/name.neuhalfen.projects.crypto.bouncycastle.openpgp/bouncy-gpg -->
```

Comment:
I'm still unsure what to do with this dependency.. since it's not on maven central nobody can build the framework without explicitly adding another repo :(

Maybe we can upload it to central, but then the groupid has to be org.ibissource (aka we need to fork it)

Reply:
I have sent an email to the owner of the project, if he would rather upload to central himself or maybe he does not care, then we can publish it. I am waiting for an anwer, depending on that I will fork and create a pipeline on Jenkins to publish.